ui/src/app/views/workspace-edit/component.html

```diff
@@ -1,17 +1,18 @@
 <h2>Create a new Workspace</h2>
+<div>* is a required field</div>
 <div class="form-area">
-  <div>
-    <label class="text-label">Name:</label>
+  <div class="name-area field">
+    <label class="text-label">Name: <span style="color: red">*</span></label>
```

Comment:
How about doing this with the `:required` CSS pseudo-class selector? https://css-tricks.com/almanac/selectors/r/required/

I'm thinking something like `<label required>` and `:required:after { content("*"); color: red; }`.